### ISSUE TITLE
The app crashes when trying to close the app

### ISSUE THREAD

**Issue by Somnus007** [alan-ai/alan-sdk-reactnative#17]
Versions:
Device OS version: Android 9
React native version: 0.66.3
@alan-ai/alan-sdk-react-native: 1.10.0

Reproduce steps:

1. Create a new react native project
2. Implement the most basic Alan integration: https://alan.app/docs/client-api/cross-platform/react-native/
3. Run the project on the localhost or run a release version of the demo app
4. Open the app and wait for Alan to finish loading without any other actions
5. Close the application, the application will crash (Not every time, please try a few more times)

![Screenshot_2021-12-14-13-34-03-361_com miui bugre](https://user-images.githubusercontent.com/31230649/145966349-aadd09dc-6ee8-4ce5-90f1-b898f447664a.png)

**Comment by snyuryev** [comment]
Hey @Somnus007 we will try to reproduce it.

**Comment by snyuryev** [comment]
Hey @Somnus007 What is device model are you using?

**Comment by Somnus007** [comment]
Redmi 6 Pro

![Screenshot_2021-12-16-09-45-45-917_com android se](https://user-images.githubusercontent.com/31230649/146293112-5ace2444-0740-43e1-aa19-8d692bf7deb8.jpg)

**Comment by Somnus007** [comment]
This screen recording video may be helpful.

https://user-images.githubusercontent.com/31230649/146862699-735c2d17-4b24-403b-aea1-ad209b1c71c2.mp4

**Comment by snyuryev** [comment]
@Somnus007 We have a new release [v 1.11.0](https://www.npmjs.com/package/@alan-ai/alan-sdk-react-native/v/1.11.0)
Please take a look - that should resolve this issue.

**Comment by Somnus007** [comment]
@snyuryev  Awesome! I just did a quick test with the new version (v1.11.0) and the app didn't crash when I tried to close it. 

But I found another problem, when developing, the Alan button will cause a problem with the hot reload of react native. Steps to reproduce the problem:
1. Create a new react native project
2. Implement the most basic Alan integration: https://alan.app/docs/client-api/cross-platform/react-native/
3. Run the project on the localhost
4. Open the app and wait for Alan to finish loading without any other actions
5. Change something in your code (such as adding a line of text) to trigger a hot reload
6. You will find that the "refreshing..." prompt will pop up twice in a row, and then the app will get stuck
